src/modules/fancyzones/FancyZonesLib/FancyZones.cpp

```diff
@@ -193,16 +187,15 @@ struct FancyZones : public winrt::implements<FancyZones, IFancyZones, IFancyZone
     bool GetSystemTheme() const noexcept;
```

Comment:
Am I missing something or these 2 (GetSystemTheme() and GetZoneColors()) are not implemented in FancyZones.cpp?